cpp/models/abm/testing_strategy.cpp

```diff
@@ -168,8 +168,8 @@ void TestingStrategy::update_activity_status(TimePoint t)
 bool TestingStrategy::run_strategy(Person::RandomNumberGenerator& rng, Person& person, const Location& location,
                                    TimePoint t)
 {
-    // Person who is in quarantine but not yet home should go home. Otherwise they can't because they test positive.
-    if (location.get_type() == mio::abm::LocationType::Home && person.is_in_quarantine()) {
+    // A Person is always allowed to go home.
```

Comment:
We should comment that this is never called if a person is not discharged from hospital or ICU.